Editor/Core/Scripts/ModuleManagement/ModuleList.cs

```diff
@@ -53,14 +39,6 @@ public static class ModuleList
             var installedModules = new Dictionary<string, string>();
             installedModules.Add(Core.name, Core.version);
 
-            foreach (ModuleInfo module in Modules)
```

Comment:
I think removing this (and WebView) from module list will cause us to lose tracking of whether WebView has been installed.

Reply:
Yeah, makes sense to have it as it is already in place